From 404fc3e2c322bc40e0c8cff5fadfb836bb357733 Mon Sep 17 00:00:00 2001
From: Alexander Shishkin <ext-alexander.shishkin@nokia.com>
Date: Fri, 21 Aug 2009 15:55:05 +0300
Subject: [PATCH 3/3] Update version to reflect our local changes.

Signed-off-by: Alexander Shishkin <ext-alexander.shishkin@nokia.com>
---
 version.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/version.h b/version.h
index cb7124f..8b8e07a 100644
--- a/version.h
+++ b/version.h
@@ -7,5 +7,5 @@
  * redistributed under the GNU Public License.
  */
 
-#define E2FSPROGS_VERSION "1.41.3"
+#define E2FSPROGS_VERSION "1.41.3.maemo0"
 #define E2FSPROGS_DATE "12-Oct-2008"
-- 
1.6.3.3

